src/ContainerInstance/test/Get-AzContainerGroup.Tests.ps1

```diff
@@ -12,12 +12,7 @@ while(-not $mockingPath) {
 . ($mockingPath | Select-Object -First 1).FullName
 
 Describe 'Get-AzContainerGroup' {
-    It 'List' {
-        {
-            Get-AzContainerGroup
```

Comment:
we'd better test each parameter set

Reply:
This is not an issue with the new API version but with the test subIds we are using. The List will always fail because the subIds we are using for testing are at max capacity for ResourceGroup creations and the response is too large. We have tried other subIds and the 'List' does not throw errors but unfortunately those cannot be used for testing. Pasting screenshot with a non-test subId. No errors. This is not an issue with our changes.
<img width="975" alt="p3" src="https://user-images.githubusercontent.com/24596325/219815965-f67ff073-bc5a-4278-b0eb-2ff8ad67776a.png">

Also, tried the List on the web page with the latest API version: https://learn.microsoft.com/en-us/rest/api/container-instances/container-groups/list?tabs=HTTP#code-try-0. No errors.

Pasting provider error with our testing subs. Even previous API versions are failing due to the response being too large.
<img width="953" alt="p1" src="https://user-images.githubusercontent.com/24596325/219816193-864534ef-3492-41e8-948c-9cba2e86afb5.png">

Reply:
We have tested with a new test sub and it worked. Will update recordings soon.

<img width="587" alt="MicrosoftTeams-image (4)" src="https://user-images.githubusercontent.com/24596325/219819401-1b1e18a6-ab45-47a5-a80a-d7f0aefd279c.png">